.github/workflows/busted.yml

```diff
@@ -1,46 +1,62 @@
+# Instead of installing busted from somewhere and using it on a code base,
+# we're actually building and running the current HEAD version on ourselves:
+# basically this is a combination bootstrap test and dogfooding. The upshot is
+# that we check both that everything works (and end-to-end test) and that the
+# current codebase checks against itself. This ensures we can fix a bug or make
+# a breaking change in busted without being blocked by our own CI using a
+# different busted verision.
+#
+# See the README.md file for examples suitable for use in other projects.
+
 name: Busted
 
-on: [push, pull_request]
+on: [ push, pull_request ]
 
 jobs:
 
   busted:
     strategy:
       fail-fast: false
       matrix:
-        luaVersion: ["5.4", "5.3", "5.2", "5.1", "luajit", "luajit-openresty"]
-        penlightVersion: ["1.9.2", "1.8.0", "1.6.0", "1.3.2"]
-    runs-on: ubuntu-20.04
+        luaVersion: [ "5.4", "5.3", "5.2", "5.1", "luajit" ] #, "luajit-openresty" ]
+        penlightVersion: [ "1.13.1", "1.9.2", "1.8.0", "1.6.0", "1.3.2" ]
+    runs-on: ubuntu-22.04
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-    - name: Setup ‘lua’
-      uses: leafo/gh-actions-lua@v8
-      with:
-        luaVersion: ${{ matrix.luaVersion }}
-    - name: Setup ‘luarocks’
-      uses: leafo/gh-actions-luarocks@v4
-    - name: Setup dependencies
-      run: |
-        sudo apt-get install libev-dev
-        luarocks install penlight ${{ matrix.penlightVersion }}
-        luarocks install --only-deps busted-scm-2.rockspec
-        luarocks install luasec
-        luarocks install moonscript
-        luarocks install copas
-        luarocks install lua-ev
-        luarocks install luacov
-    - name: Cache Lua machinery
-      uses: actions/cache@v2
-      with:
-        path: |
-          .install
-          .lua
-          .luarocks
-        key: lua-${{ matrix.luaVersion }}-${{ matrix.penlightVersion }}-${{ hashFiles('busted-scm-2.rockspec', '.github/workflows/busted.yml') }}
-    - name: Build
-      run: |
-        luarocks remove busted --force ||:
-        luarocks make
-    - name: Run tests
-      run: busted -v
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup ‘lua’
+        uses: leafo/gh-actions-lua@v9
+        with:
+          luaVersion: ${{ matrix.luaVersion }}
+
+      - name: Setup ‘luarocks’
+        uses: leafo/gh-actions-luarocks@v4
+
+      - name: Setup dependencies
+        run: |
+          sudo apt-get install libev-dev
+          luarocks install penlight ${{ matrix.penlightVersion }}
+          luarocks install luasec
+          luarocks install moonscript
+          luarocks install copas
+          luarocks install lua-ev
+          luarocks install luacov
+          luarocks install --deps-only busted-scm-2.rockspec
+
+      - name: Build ‘busted’ (bootstrap)
+        run: |
+          luarocks make
+
+      - name: Run ‘busted’ (dogfood)
+        run: busted -c -v
```

Comment:
```suggestion
        # disable the paths to force use of installed rockspec
        run: busted -c -v --lpath="" --cpath="" --Xoutput "--color"'
```

Reply:
the coverage option in `.busted` should be removed

Reply:
This project doesn't even have a `.busted`. Per explanation in [this comment](https://github.com/lunarmodules/say/pull/16#discussion_r952379886) I'm not in favor of adding one either but of course if it did the coverage option should not be enabled.